src/content/nft/index.md

```diff
@@ -273,6 +274,24 @@ Companies offering fake certificates for university degrees are reportedly a bil
 
 <Divider />
 
+### Tokengating {#tokengating}
+
+Tokengating is a way of restricting access to something and using tokens as a way to unlock access. These things can vary hugely based on the platform but popular examples are gated content, private chat servers, and, in the world of ecommerce, exclusive products.   
+
+A platform that does tokengating will typically ask you to connect your wallet to prove you own the required tokens. If you have the tokens you need, you'll get access. If not, the thing will remain gated. NFTs are a great way to do this because of their uniqueness – you can't fake ownership to get the thing. 
```

Comment:
```suggestion
A platform that does tokengating will typically ask you to connect your wallet to prove you own the required NFT. If you have the NFT you need, you'll get access. If not, the thing will remain gated. NFTs are a great way to do this because of their uniqueness – you can't fake ownership to get the thing. 
```